### PR TITLE
Fixed "Varified" typo

### LINKED ISSUE #250: Typo in email verification in Sidebar
**Describe the issue**
After login, in the sidebar, if the email is not verified, it shows "NOT VARIFIED". Here clearly the spelling of verified is a typo. It should've been "NOT VERIFIED" instead.

**To Reproduce**
1. Login
2. View Sidebar
3. See error

**Expected behavior**
Instead of "VARIFIED", "VERIFIED", should be present.

**Screenshots**

Screenshot 2022-02-09 at 4 59 23 PM

**Desktop:**
 - OS: [MacOS]
 - Browser [Brave]
 - Version [11.6]

**Additional context**

### PR DESCRIPTION
**Pull request by PrerakMathur20**
# Description
In several files the key word "Varified" was present, which is supposed to be "Verified". The modified files are -

- ```React-frontend/src/components/Admin/ShowMembers.js```
- ```React-frontend/src/components/Utils/VerifyButton.js```
-  ```flask-backend/api/models/base_user.py```
- ```flask-backend/api/schema/admin.py```
- ```flask-backend/api/schema/extractor.py```
- ```flask-backend/api/schema/management.py```
- ```flask-backend/api/userAuthentication/auth.py```
- ```flask-backend/migrations/versions/989dbc01a9b0_.py```

Rather than just fixing the "Varified" term in only the ```React-frontend/src/components/Admin/ShowMembers.js``` file's print message, I found out there are several functions called by the name ```varified``` and ```isVarified```. So I went on, and fixed the typo there as well, and obviously all of their Occurrence in other relevant code as well.

Fixes #250 

## Type of change

- [x] This Change increases code readability 
- [x] Fixes Frontend Type

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] ```Verified``` appears everywhere now, which earlier was ```varified```.
- [x] Created new account, signed in and attempted verifying email. No Issues found.

**Test Configuration**:

- Firmware version:
- Hardware:
- Toolchain:
- SDK:

##Screenshot 
Also, include screenshots for the verification and reviewing purpose.

**Before**
OpenMF_Before

**After**
OpenMF_After

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation **(N/A)**
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works **(N/A)**
- [ ] New and existing unit tests pass locally with my changes **(N/A)**
- [ ] Any dependent changes have been merged and published in downstream modules **(N/A)**

### CODE REVIEW

File: flask-backend/api/schema/management.py [hunk 1]
```diff
@@ -10,4 +10,4 @@ class ManagementSchema(ma.Schema):
     Schema for admin model
     """
     class Meta:
-        fields = ('name', 'email', 'public_id', 'varified', 'created_on', 'updated_on', 'role')
\ No newline at end of file
+        fields = ('name', 'email', 'public_id', 'vefified', 'created_on', 'updated_on', 'role')
```
Comment (swapnalshahil) [inline flask-backend/api/schema/management.py]: Check here!

Comment (PrerakMathur20) [reply flask-backend/api/schema/management.py]: My apologies for letting this typo slide through. I'll fix this right away.

**Comment by PrerakMathur20** [comment]
@swapnalshahil I have made the required changes and fixed the error. Can you please take a look at this, and let me know If there's anything else in the way of this PR getting merged?

**Comment by PrerakMathur20** [comment]
@charithccmc 
Can you please review this PR? And if there's anything that is stopping it from merging?

**Pull request merged**